README.md

```diff
@@ -72,17 +76,108 @@ sha256sum: WARNING: 19 lines are improperly formatted
 
-```bash
-./gradlew run -PmainClass=co.rsk.federate.FederateRunner
+- Set String generator = "federator1";
+
+- Run the Class to generate a private key
+
+- Create a file **reg1.key** with the private key as the content
+
+- Add permission to the file by running **chmod 400 reg1.key**
+
+
+8. Creating the signers configuration. **(For Local test / regtest only)**
+
+Create a file **regtest-fed.conf** with the config below
+
+### Signer's configurations
+```
+federator {
+    # keep it false if you don't want to interact with bitcoin
+    enabled = true
+    
+    signers {
+       BTC {
+          type = "keyFile"
+          path = "A/PATH/TO/YOUR/BTC-KEY.key"
+       }
+       RSK {
+          type = "keyFile"
+          path = "A/PATH/TO/YOUR/RSK-KEY.key"
+       }
+       MST {
+          type = "keyFile"
+          path = "A/PATH/TO/YOUR/MST-KEY.key"
+       }
+    }
+    # peers for the bitcoin network
+    bitcoinPeerAddresses = [
+        "127.0.0.1:18444"
+    ]
+}
 ```
 
+9. Steps to import and configure the project.
+
 *To import the project to [IntelliJ IDEA](https://www.jetbrains.com/idea/download):*
 
-You can import the project to your IDE. For example, [IntelliJ IDEA Community Edition](https://www.jetbrains.com/idea/). To import go to `File | New | Project from existing sources...` Select `federate-node/build.gradle` and import. After building, run `co.rsk.federate.FederateRunner`
+You can import the project to your IDE. For example, [IntelliJ IDEA Community Edition](https://www.jetbrains.com/idea/). To import go to `File | New | Project from existing sources...` Select `federate-node/build.gradle` and import.
+
+- Create a New Application configuration with name **FedRunner**
+
+**VM options**
+
+- -Drsk.conf.file=/<PATH-TO-CONF-FILE>/regtest-fed.conf
+- -Dlogback.configurationFile=/<PATH-TO-LOG-FILE>/logback.xml
+- -Dblockchain.config.hardforkActivationHeights.iris300=0
+
+**Program Arguments**
+
+- --regtest --reset
+
+**Module**
+
+- -cp federate-node.main
+
+**Main Class**
+
+- co.rsk.federate.FederateRunner
+
+Then clean and build project using **./gradle clean build**
+
+10. Install and run Bitcoind
+
+   In order to install Bitcoind, you can follow these steps. We will use version ``` 0.18.0 ```, but it can be any other valid one (i.e ``` 0.17.1 ```).
+
+```bash
+   curl -O https://bitcoin.org/bin/bitcoin-core-0.18.0/bitcoin-0.18.0-osx64.tar.gz
+   tar -zxf bitcoin-0.18.0-osx64.tar.gz
+   sudo mkdir -p /usr/local/bin
+   sudo cp bitcoin-0.18.0/bin/bitcoin* /usr/local/bin/.
+   rm -rf bitcoin-0.18.0*
+   As a validation, you can run *bitcoind -daemon*. Run *bitcoin-cli stop* afterwards.
+```
+Create the scripts below as a file and its content:
+
+**bitcoin-node.sh**
+```bash
+#!/bin/bash
+
+bitcoind -regtest -printtoconsole -server -rpcuser=rsk -rpcpassword=rsk -rpcport=18332 -txindex -debug=net --deprecatedrpc=signrawtransaction -addresstype=legacy -deprecatedrpc=accounts -deprecatedrpc=generate
```

Comment:
```suggestion
bitcoind -regtest -printtoconsole -server -rpcuser=rsk -rpcpassword=rsk -rpcport=18332 -txindex -debug=net -deprecatedrpc=signrawtransaction -addresstype=legacy -deprecatedrpc=accounts -deprecatedrpc=generate
```

Reply:
Maybe we could add also `-datadir=PATH/TO/DATA/DIRECTORY`

Not sure if it's required or not